test/test.address.js

```diff
@@ -1,6 +1,6 @@
 /// <reference path="../chance.js" />
 /// <reference path="../node_modules/underscore/underscore-min.js" />
-/// <reference path="lib/phone_number_js/dist/phoneNumber.min.js" />
+/// <reference path="test/lib/phone_number_js/dist/phoneNumber.min.js" />
```

Comment:
This change is causing the unit tests to fail because this path is relative to the test directory. Can you please change it back so the tests pass? Thanks!